### PR TITLE
🐛 whoops - local change sneaked in

### PR DESCRIPTION
**Pull request by dims**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: config/default/manager_image_patch.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
+      - image: gcr.io/vincepri-dev/cluster-api-gcp-controller-amd64:dev
```
Comment (vincepri) [inline config/default/manager_image_patch.yaml]: :master?

Comment (detiber) [inline config/default/manager_image_patch.yaml]:
```suggestion
```

Comment (vincepri) [reply config/default/manager_image_patch.yaml]: This shouldn't be removed entirely, we should use the staging bucket as it was before

Comment (detiber) [reply config/default/manager_image_patch.yaml]: The image is listed twice

**Pull request closed without merging**